config/sites/fiwiki.yml

```diff
@@ -99,9 +99,13 @@ templates:
                 name: sparql  # as in {{ ukb criterion | sparql }}
                 params:
                     query: query    # as in {{ ukb criterion | sparql | query=... }}
+<<<<<<< HEAD
```

Comment:
Merge conflict that needs to be resolved.

Reply:
Thanks, done! Not sure how I managed that… 😅